server/models/index.js

```diff
@@ -0,0 +1,42 @@
+const { Sequelize } = require("sequelize");
+const BookModel = require("./book");
+const AuthorModel = require("./author");
+require("dotenv").config({
+  path: "../.env",
+});
+
+const sequelize = new Sequelize(process.env.POSTGRES_URI);
+
+async function createDatabaseIfNotExists() {
```

Comment:
This is going to add a query every time sever starts? Would this be efficient in long run? Also should we enforce db name to always be booksdb?

Reply:
No it will not be efficient in the long run and this should be moved out of this